plugins/azure_ad_admin/help.md

```diff
@@ -50,6 +51,40 @@ Example input:
 
 ### Actions
 
+#### Change User Password
+
+This action is used to change a user password by an administrator with appropriate permissions.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|new_password|password|None|True|The new password|None|newPassword|
```

Comment:
Can you please make sure that the logs do not show up the password in the product? Both for valid and error cases.

Reply:
<img width="430" alt="Screenshot 2022-11-22 at 10 31 28" src="https://user-images.githubusercontent.com/99184394/203278550-b4c3f129-a9a4-4a5f-8fb1-6402d4963232.png">
<img width="802" alt="Screenshot 2022-11-22 at 10 33 58" src="https://user-images.githubusercontent.com/99184394/203278586-f1714e7d-5171-4d4d-9209-f2875bc6b4a6.png">

We do not provide passwords anywhere. Even when plugin failed